package.json

```diff
@@ -15,6 +15,7 @@
     "carbon-icons": "^7.0.7",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-google-login": "^5.1.22",
```

Comment:
You aren't using this anywhere. You should remove this since it could cause merge conflict later.